compiler/lib/src/main/scala/util/Error.scala

```diff
@@ -195,8 +195,12 @@ sealed trait Error {
       case SemanticError.MissingConnection(loc, matchingLoc) =>
         Error.print (Some(loc)) ("no match for this connection")
         printMatchingLoc(matchingLoc)
+      case SemanticError.MissingPortMatching(loc) =>
+        Error.print (Some(loc)) ("unmatched connection must go from or to a matched port")
       case SemanticError.MissingPort(loc, specMsg, portMsg) =>
         Error.print (Some(loc)) (s"component with $specMsg must have $portMsg")
+      case SemanticError.MultipleConnectionsAtPortIndex(portIndexString) =>
+        Error.print (None) (s"multiple connections appear at port indices $portIndexString")
```

Comment:
I think this error message should not refer to indices (plural). The common case (and maybe the only case?) is a single index.

Reply:
True! I updated the wording in the error to be "index" vs. "indices".